subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_coap

```diff
@@ -52,7 +52,7 @@ endif # WIFI
 config NRF_CLOUD_COAP_DOWNLOADS
 	bool "Download files using CoAP instead of HTTP"
 	depends on NRF_CLOUD_COAP
-	select EXPERIMENTAL
+	default y if NRF_CLOUD_FOTA_POLL || NRF_CLOUD_PGPS
```

Comment:
This causes compilation of modem_shell with P-GPS and CoAP to fail.

Please see PR #17899 description. That will disable `CONFIG_NRF_CLOUD_COAP_DOWNLOADS` so that we would get our CI pass for P-GPS+CoAP configuration.

I guess the issue is only visible on MOSH because others using CONFIG_COAP_EXTENDED_OPTIONS_LEN_VALUE doesn't use CoAP with PGPS or FOTA.